tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "baseUrl": "./",
     "paths": {
       "components/*": ["src/components/*"],
```

Comment:
maybe we want to use a symbol or something to indicate this path is an alias, something like `~`. No need to change this though

Reply:
Agree!